README.md

```diff
@@ -1,11 +1,16 @@
-# required PHP Coding Standard
+# required Coding Standards
 
-required coding standard for PHP, inspired by the [Human Made Coding Standards](https://github.com/humanmade/coding-standards) project.
+The required coding standards represent the best practices for enabling a consistent code style across the team. The standards do not only contain stylistic rules. They are also defined to prevent potential security vulnerabilities or to leverage more modern syntax for enhanced performance.  
+This project is inspired by the [Human Made Coding Standards](https://github.com/humanmade/coding-standards) project.
 
 For the JavaScript coding standard head over to [@wearerequired/eslint-config](packages/eslint-config).  
 For the (S)CSS coding standard head over to [@wearerequired/stylelint-config](packages/stylelint-config).
 
-## Setup
+Please contact the handbook if you need help with [setting up the coding standards in your environment](https://handbook.required.com/development/coding-standards/).
```

Comment:
As this is a public repo it would be good to mention that this is an internal link.